tests/manage/z_cluster/cluster_expansion/test_node_expansion.py

```diff
@@ -16,19 +15,25 @@ class TestAddNode(ManageTest):
     """
     Automates adding worker nodes to the cluster while IOs
     """
-
-    def test_add_node(self):
+    @aws_platform_required
+    def test_add_node_aws(self):
         """
         Test for adding worker nodes to the cluster while IOs
         """
         dt = config.ENV_DATA['deployment_type']
         if dt == 'ipi':
             machines = machine_utils.get_machinesets()
-            logger.info(f'The worker nodes number before {len(helpers.get_worker_nodes())}')
+            logger.info(f'The worker nodes number before expansion {len(helpers.get_worker_nodes())}')
             for machine in machines:
                 add_new_node_and_label_it(machine)
-            logger.info(f'The worker nodes number after {len(helpers.get_worker_nodes())}')
+            logger.info(f'The worker nodes number after expansion {len(helpers.get_worker_nodes())}')
 
         else:
-            pytest.skip("UPI not yet supported")
-        # ToDo run IOs
+            logger.info(f'The worker nodes number before expansion {len(helpers.get_worker_nodes())}')
+            new_nodes = 3
```

Comment:
probably you can use the len function here as well before and after the start of test to make it generic.

Reply:
ack